engineioxide/src/engine.rs

```diff
@@ -104,6 +108,7 @@ where
     /// Otherwise it will wait for the next packet to be sent from the socket
     pub(crate) async fn on_polling_http_req<B>(
         self: Arc<Self>,
+        protocol: ProtocolVersion,
```

Comment:
Here the `ProtocolVersion` param could be remove with feature flags..

Reply:
This has changed recently. I'll await a new review as discussed below.